### PR TITLE
[DEV-195] fix php documentation

### PR DESCRIPTION
**Pull request by joelmendez1**

### CODE REVIEW

File: server/controllers/article/edit.php
Comment (ivandiazwm) [inline server/controllers/article/edit.php:27]: I think this exists in `Controller.php` in `uploadImages`

File: server/controllers/ticket/add-custom-response.php
Comment (ivandiazwm) [inline server/controllers/ticket/add-custom-response.php:22]: `INVALID_TITLE` is missing here

File: server/controllers/ticket/comment.php
Comment (ivandiazwm) [inline server/controllers/ticket/comment.php:29]: same here for INVALID_FILE

File: server/controllers/ticket/create.php
Comment (ivandiazwm) [inline server/controllers/ticket/create.php:35]: here too

**Review by ivandiazwm — CHANGES_REQUESTED** [review]

**Comment by mredigonda** [comment]
Are there any updates on this PR?

If you have any please upload them and re-request review from Ivan 👍🏼.

File: server/controllers/system/init-settings.php
Comment (mredigonda) [inline server/controllers/system/init-settings.php:28]: This also isn't being used.

File: server/controllers/system/stats.php
Comment (mredigonda) [inline server/controllers/system/stats.php:23]: Remove it from `ERRORS.php` too.

File: server/controllers/ticket/comment.php
Comment (mredigonda) [reply server/controllers/ticket/comment.php:29]: I guess it's used implicitly by the `FileUploader`? In that case (if this path can return that error) then let's leave it.

File: server/controllers/ticket/edit-title.php
Comment (mredigonda) [inline server/controllers/ticket/edit-title.php:22]: Seems we are also using `INVALID_TICKET` error here.

**Review by mredigonda — COMMENTED** [review]
I noticed some small details to complete with this PR, after that, we can merge!

**Review by mredigonda — APPROVED** [review]
LGTM 🎉

**Pull request merged**
